.gitmodules

```
@@ -1,6 +1,4 @@
-[submodule "extern/go-bls-sigs"]
-	path = extern/go-bls-sigs
-	url = https://github.com/filecoin-project/go-bls-sigs.git
-[submodule "extern/go-sectorbuilder"]
-	path = extern/go-sectorbuilder
-	url = https://github.com/filecoin-project/go-sectorbuilder
+[submodule "extern/filecoin-ffi"]
+	path = extern/filecoin-ffi
+	url = git@github.com:filecoin-project/filecoin-ffi
```

Should be
```suggestion
	url = https://github.com/filecoin-project/filecoin-ffi.git
```
So we don't force people building from source to have ssh installed